src/styles.css

```
@@ -23,6 +23,42 @@
   --input-bg:#1A1108;--tag-bg:#0F0A05;--modal-bg:#1A1108;--an-bg:#0F0A05;
   --skeleton-base:#2a2a3a;--skeleton-shine:#33334a;
 }
+[data-theme="sepia"]{
+  --bg:#F5EEDC;--white:#FFFBF0;
+  --orange:#D45800;--orange-deep:#9D4300;--orange-pale:#EBE3CE;--orange-mid:#DCCCAB;--orange-border:#CBA983;
+  --ink:#3A2010;--ink2:#5A3015;--ink3:#7A5030;--muted:#9A6C4A;
+  --border:#DCCCAB;--border2:#E5D6BB;
+  --green:#006C47;--red:#9B2C2C;--blue:#1E40AF;--purple:#553C9A;
+  --shadow:0 1px 4px rgba(58,32,16,.08),0 6px 20px rgba(58,32,16,.06);
```

Stylelint (17.12.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)